code/modules/antagonists/cult/cult_structures.dm

```diff
@@ -54,6 +54,25 @@
 	else
 		..()
 
+/*
+/obj/structure/destructible/cult/set_anchored(anchorvalue)
+	. = ..()
+	if(isnull(.))
+		return
+	update_icon()
```

Comment:
what is this

Reply:
something me of 3 months ago forgot to test

removed in 3814797